front-end/legeno-around-here/src/components/api/API.js

```diff
@@ -650,15 +661,37 @@ export const deletePost = (accessToken, postId, history) => {
       alert('정상적으로 삭제되었습니다!');
       history.push('/home');
     })
-    .catch((error) => {
+    .catch(async (error) => {
+      await redirectLoginWhenUnauthorized(error, history);
       const errorResponse = error.response.data;
       alert(errorResponse.errorMessage);
-      redirectLoginWhenUnauthorized(error, history);
     });
 };
 
+export const withdraw = (accessToken) => {
+  const config = {
+    headers: {
+      'X-Auth-Token': accessToken,
+    },
+  };
+  axios
+  .delete(DEFAULT_URL + '/users/me', config)
+  .then(async (response) => {
+    await removeAccessTokenCookie();
+    alert('탈퇴 완료되었습니다. 그동안 이용해주셔서 감사합니다!');
+    document.location.href = '/';
```

Comment:
```suggestion
    history.push('/home');
```
해당 부분도 history를 파라미터로 받아와서 push해주셔야 SPA가 될 것 같아요 :)

Reply:
수정했습니다!!